src/Resources/assets/controllers/table_select_controller.js

```diff
@@ -90,7 +90,14 @@ export default class extends Controller {
             body: formData
         }) .then(async response => {
             if (response.status == 200) {
-               window.location.href = response.url;
+                let data = await response.json();
+                if (data.reload) {
+                    window.location.reload();
+                }
+                if (data.redirect) {
+                    window.location.href = data.redirect;
+                }
+                console.warn('Batch Action Controller should return json data with reload or redirect information');
```

Comment:
The console warn should only be run when `data.reload` or `data.redirect` is missing. 

The redirect resp. reload takes roughly a second and during this time you see the warning.

Reply:
Done